nav2_map_server/src/map_server/map_server_core.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2020 Shivam Pandey
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "nav2_map_server/map_server_core.hpp"
+
+namespace nav2_map_server
+{
+
+template<class mapT>
+MapServer<mapT>::MapServer()
```

Comment:
This doesnt look used (mentioned in the header file of the same object) -- remove if not used